modules/EnsEMBL/Web/SpeciesDefs.pm

```diff
@@ -1541,11 +1574,18 @@ sub production_name_mapping {
 ### As the name said, the function maps the production name with the species URL, 
 ### @param production_name - species production name
 ### Return string = the corresponding species.url name which is the name web uses for URL and other code
+### Fall back to production name if not found - mostly for pan-compara
   my ($self, $production_name) = @_;
+  my $mapping_name = $production_name;
   
   foreach ($self->valid_species) {
-    return $self->get_config($_, 'SPECIES_URL') if($self->get_config($_, 'SPECIES_PRODUCTION_NAME') eq lc($production_name));
+    if ($self->get_config($_, 'SPECIES_PRODUCTION_NAME') eq lc($production_name)) {
```

Comment:
sorry one more where we do not need the `lc`

Reply:
Well spotted!